cumulusci/core/dependencies/dependencies.py

```diff
@@ -329,8 +329,14 @@ def flatten(self, context: BaseProjectConfig) -> List[Dependency]:
             )
         )
 
-        # Deploy the project, if unmanaged.
-        if not managed:
+        if not self.managed_dependency:
```

Comment:
Would it be reasonable to rename `managed_dependency` to `packaged_dependency`?

Reply:
Sure. Done.